lte/gateway/c/oai/include/mme_config.h

```diff
@@ -332,6 +348,8 @@ typedef struct mme_config_s {
 
   restricted_plmn_config_t restricted_plmn;
 
+  blocked_imei_config_t blocked_imei;
```

Comment:
it is better to define this as a hash table. you need to define macros for converting IMEI (fully or only the TAC part) into an integer key.